src/client/conn.rs

```diff
@@ -75,6 +75,7 @@ pub struct Builder {
     h1_writev: bool,
     h1_title_case_headers: bool,
     h1_read_buf_exact_size: Option<usize>,
+    max_buf_size: Option<usize>,
```

Comment:
Since it's specific to HTTP1, what if the name were prefixed with `h1_`, like the others?